CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Fallback to user's log path if the default log path (`$PIPX_HOME/logs`) is not writable to aid with pipx being used for multi-user (e.g. system-wide) installs of applications
 - Fix wrong interpreter usage when injecting local pip-installable dependencies into venvs
 - add pre-commit hook support
+- Don't use `repr` for paths in console output 
```

Comment:
I don't think this is worth a changelog entry.

Reply:
I would add an entry, but not like this. It should say something about not showing double backslashes on Windows instead.

Reply:
Done